packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -92,6 +94,10 @@ abstract class Scrollable extends StatefulWidget {
   /// Called whenever this widget stops scrolling.
   final ScrollListener onScrollEnd;
 
+  final VoidCallback onDragStart;
+
+  final VoidCallback onDragEnd;
```

Comment:
I've removed the dragStart/dragEnd plumbing. For now that means that a drag gesture that occurs while an overscroll fling is underway (and otherwise not observable) doesn't expose the overscroll indicator.  To get the right behavior:

- Create a variation of OverScrollBehavior that ignores fling velocity when the scrollable is at an end and a fling would cause overscroll. Not sure exactly what to call this. Maybe it's a feature of OverscrollBehavior.

- Create an inherited ScrollTheme widget. Remove the Scrollable createScrollBehavior() factory method; lookup scroll behaviors instead. Also lookup ScrollableListPainters.

- Remove the clampOverscrolls property from ScrollableList.

All of this in one or more separate PRs.